XenAdmin/SettingsPanels/NRPEEditPage.resx

```diff
@@ -117,224 +117,151 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
-  <data name="NRPETableLayoutPanel.Anchor" type="System.Windows.Forms.AnchorStyles, System.Windows.Forms">
-    <value>Top, Left, Right</value>
-  </data>
   <assembly alias="mscorlib" name="mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
   <data name="NRPETableLayoutPanel.AutoSize" type="System.Boolean, mscorlib">
     <value>True</value>
   </data>
   <data name="NRPETableLayoutPanel.ColumnCount" type="System.Int32, mscorlib">
     <value>1</value>
   </data>
-  <data name="PoolTipsTableLayoutPanel.Anchor" type="System.Windows.Forms.AnchorStyles, System.Windows.Forms">
-    <value>Top, Left, Right</value>
-  </data>
-  <data name="PoolTipsTableLayoutPanel.AutoSize" type="System.Boolean, mscorlib">
+  <data name="EnableNRPECheckBox.AutoSize" type="System.Boolean, mscorlib">

```

Comment:
In this file I have made some layout tweaks. The main things to note are:
- I removed the info at the bottom and integrated the information instead in the top label.
- I moved the checkbox outside the groupbox because it is not part of the configuration, but is rather used to switch the feature on/off.
- added hotkey to the controls so one can use the UI only with the keyboard.
- in general the aim is to modify the default values of margins, padding etc. as little as possible.